File: jplan/src/main/java/com/jplan/jplan/models/requests/TimeCardRequest.java
package com.jplan.jplan.models.requests;

import java.sql.Date;

import lombok.Data;

@Data
public class TimeCardRequest {

    private String username;

    private Date start_time;

    private Date stop_time;

    private boolean active;

    private boolean edited;

    private String projectId;

    private String comment;
}

File: client/j_plan/src/components/models/TimeCardRequest.ts
export default class TimeCardRequest {
    username: string;
    comment: string;
    active: boolean;
    edited: boolean;
    startTime: string;
    endTime: string;
    
    constructor(username: string, comment: string, active:boolean, edited: boolean, startTime: string, endTime: string){
        this.username = username;
        this.comment = comment;
        this.active = active;
        this.edited = edited;
        this.startTime = startTime;
        this.endTime = endTime;
    }
}